src/System.Management.Automation/engine/remoting/commands/PSRemotingCmdlet.cs

```diff
@@ -856,8 +880,16 @@ internal SSHConnection[] ParseSSHConnectionHashTable()
                     if (paramName.Equals(ComputerNameParameter, StringComparison.OrdinalIgnoreCase) || paramName.Equals(HostNameAlias, StringComparison.OrdinalIgnoreCase))
                     {
                         var resolvedComputerName = ResolveComputerName(GetSSHConnectionStringParameter(item[paramName]));
-                        ValidateComputerName(new string[] { resolvedComputerName });
```

Comment:
Do we want to no longer validate the host name?

Reply:
I suppose I can call ValidateComputerName after extracting the hostname